README.md

```diff
@@ -23,3 +23,40 @@ command-line tools in bin/, but the recommended interface is the
 memtrace viewer, which lives at:
 
     https://github.com/janestreet/memtrace_viewer
+
+## Installation
+These instructions are for using statmemprof with OCaml 5.* based on
+[ocaml#12923](https://github.com/ocaml/ocaml/pull/12923) which restores statmemprof for multicore plus
+[ocaml#12963](https://github.com/ocaml/ocaml/pull/12963) which fixes some runtime parameter queries.
+
+``` shell
+# Setup a new Blank switch
+opam switch create statmemprof --empty
+
+# Install OCaml trunk with #12923 + #12963
+opam pin add ocaml-variants git+https://github.com/tmcgilchrist/ocaml#caml_runtime_parameters
```

Comment:
Now that statmemprof is merged upstream, can we use simpler instructions here?

Reply:
Updated. Simplest version is using `5.3.0+trunk` plus this PR.